src/Service/Azure.DataApiBuilder.Service.csproj

```diff
@@ -77,6 +77,11 @@
         <PackageReference Include="System.CommandLine" />
         <PackageReference Include="System.IO.Abstractions" />
         <PackageReference Include="ZiggyCreatures.FusionCache" />
+        <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Condition="'$(TargetFramework)' == 'net8.0'" />
+        <PackageReference Include="OpenTelemetry.Extensions.Hosting" Condition="'$(TargetFramework)' == 'net8.0'" />
+        <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Condition="'$(TargetFramework)' == 'net8.0'" />
+        <PackageReference Include="OpenTelemetry.Instrumentation.Http" Condition="'$(TargetFramework)' == 'net8.0'" />
+        <PackageReference Include="OpenTelemetry.Instrumentation.Runtime" Condition="'$(TargetFramework)' == 'net8.0'" />
```

Comment:
If you wanted to simplify this, you could move them into their own `ItemGroup` and put the condition on the `ItemGroup` rather than on each `PackageReference`.